docs/intl.md

```diff
@@ -0,0 +1,155 @@
+# Spark Regional Date Time Support
+
+## Spark Components support regional date functionality
+
+- Spark components have been updated to support regional date functionality in latest versions of **V3** and **V4**.
+- The new updates in Spark are internally feature toggled so that we can release the new date/time regional at the same time for all apps
+- There are two methods for enabling the new regional date features in Spark components: a global variable, and a feature toggle in the form of a cookie
+  o To test the new Spark component regional date formatting in your application, set the global variable on the window object:
+
+      ```
+      window['GUX_OPTIONS_enableRegionalDates']= true
+      ```
+
+## Feature Toggling
+
+- All regional date functionality should be feature toggled. We want to avoid confusing customers with different date time behavior across applications
+- The Spark component functionality is internally feature toggled and when it is time to GA this feature, will be globally enabled via a cookie set in web directory
```

Comment:
What is `GA`? I'm probably just not familiar yet with this.

Reply:
Thoughts on this wording? 

`The Spark component functionality is internally feature toggled. It will be globally enabled via a cookie set in web directory when it is time to GA this feature`